tests/console/force_cron_run.pm

```diff
@@ -30,18 +30,25 @@ sub settle_load {
 
 sub run {
     select_console 'root-console';
-
     # show dmesg output in console during cron run
     assert_script_run "dmesg -n 7";
 
     # Make sure there's no load before we trigger one via cron.
     settle_load;
     my $before = time;
-    assert_script_run "bash -x /usr/lib/cron/run-crons", 1000;
+    # run cron jobs or systemd timers which can affect system performance and mask systemd timers later
+    assert_script_run('find /etc/cron.{daily,weekly,monthly} -type f -executable -exec echo cron job: {} \; -exec {} \;');
+    assert_script_run(
+        'for i in $(systemctl list-units --type=timer --state=active --no-legend | awk \'{ print $1 }\' | sed \'s/.timer//\'); do
+    echo "Triggering systemd timed service $i" && systemctl start $i ; done'
```

Comment:
Just immediately mask the service here as well so then we do not even have any duplication further down